src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -452,7 +452,79 @@ ORDER BY attnum;
 (29 rows)
 
 \c - - - :master_port
+-- check we don't intercept DDL commands if propagation is turned off
+SET citus.enable_ddl_propagation to false;
+-- table rename statement can be performed now
+ALTER TABLE lineitem_alter RENAME TO lineitem_renamed;
+-- verify rename is performed
+SELECT relname FROM pg_class WHERE relname = 'lineitem_alter' or relname = 'lineitem_renamed';
+     relname      
+------------------
+ lineitem_renamed
+(1 row)
+
+-- revert it to original name
+ALTER TABLE lineitem_renamed RENAME TO lineitem_alter;
+-- this column is added to master table and not workers
+ALTER TABLE lineitem_alter ADD COLUMN not_added_column int;
+-- verify newly added column is not present in a worker shard
+\c - - - :worker_1_port
+SELECT not_added_column FROM lineitem_alter_103000 LIMIT 0;
+ERROR:  column "not_added_column" does not exist
+LINE 1: SELECT not_added_column FROM lineitem_alter_103000 LIMIT 0;
+               ^
+\c - - - :master_port
+-- ddl propagation flag is reset to default, disable it again
+SHOW citus.enable_ddl_propagation;
+ citus.enable_ddl_propagation 
+------------------------------
+ on
+(1 row)
+
+SET citus.enable_ddl_propagation to false;
+-- following query succeeds since it accesses an previously existing column
+SELECT l_orderkey FROM lineitem_alter LIMIT 0;
```

Comment:
> I didn't understand the reason for this query. Could you explain a bit more?

Ohh, you mean if we run a query on the new column, it should error on worker nodes?